docs/community.md

```diff
@@ -1,102 +1,113 @@
+| ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- | --------------------------------------------------------------- | ----------------- |
+| <center>[Algorand QRCode Generator](https://github.com/emg110/algorand-qrcode)</center>              | A comprehensive javascript module with a complete set of tools to generate an standard Algorand URI (RFC 3986) and QR code, exportable to SVG, PNG and UTF8. Exported media types are File, UTF8 text and DataURL. Works in Terminal, Node and modern browsers. Contains a full set of examples including API server, static server and web form QR generator. Includes a full featured CLI to generate QR codes in terminals too. See solution [here](https://developer.algorand.org/solutions/algorand-qr-code-generator-javascript/). |
+| <center>[Algorand Wallet Manager](https://github.com/CiottiGiorgio/Algorand-Wallet-Manager)</center> | This Python application is a graphic interface for the user who owns or have access to an Algorand node and would like to issue operations through a GUI rather than CLI. (i.e.: manage wallets and addresses, send transactions, save a contact list)                                                                                                                                                                                                                                                                                   |
+| <center>[How big is an Algorand Archival Node?](https://howbigisalgorand.com/)</center>              | Dashboard about the size of an Algorand Archival Node. Data is live. Thanks [Harsh Baid](https://forum.algorand.org/u/harshbaid/summary)!                                                                                                                                                                                                                                                                                                                                                                                                |     | <center>[Sandbox](https://github.com/algorand/sandbox)</center> | Algorand Sandbox. |
```

Comment:
Remove "Thanks [Harsh Baid](https://forum.algorand.org/u/harshbaid/summary)!" (looks like a comment, not description)

Reply:
This is resolved in this commit: https://github.com/algorand/docs/pull/423/commits/2a9cad155e6eca0b606cae8f108e6d631db6d45d